app/code/community/Sailthru/Email/controllers/ContentController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+class Sailthru_Email_ContentController extends Mage_Adminhtml_Controller_Action {
+
+    const MAX_ITEM_POST = 350;
+
+    /**
+     * <magento_uri>/content/bulk endpoint action
+     */
+    public function bulkAction() {
+        $data = $this->getRequest()->getPost();
+        if ($data and array_key_exists('product', $data) and
+            array_key_exists('massaction_prepare_key', $data) and $data['massaction_prepare_key'] == 'product' and
+            array_key_exists('store', $data) and intval($data['store']) > 0) {
+            $this->_processItems($data['product'], $data['store']);
+        } else {
+            Mage::getSingleton('adminhtml/session')->addError("Request to bulk update Sailthru was malformed. Please try again.");
```

Comment:
"was malformed" doesn't give them much info, would "Request to bulk update Sailthru failed" be more appropriate? What would they do to fix it?

Reply:
No idea why that error would occur. Shouldn't happen. Hence debug POST info available.